README.md

```diff
@@ -20,95 +20,50 @@ More information about releases in Octopus Deploy:
 Incorporate the following actions in your workflow to create a release in Octopus Deploy using an API key, a target instance (i.e. `server`), and a project:
 
 ```yml
+env:
+  OCTOPUS_HOST: ${{ secrets.SERVER }}
+
 steps:
   # ...
   - name: Install Octopus CLI 🐙
     uses: OctopusDeploy/install-octopus-cli-action@v1
     with:
       version: latest
+
   - name: Create a release in Octopus Deploy 🐙
-    uses: OctopusDeploy/create-release-action@v1
+    uses: OctopusDeploy/create-release-action@v2
     with:
       api_key: ${{ secrets.API_KEY }}
-      project: 'Projects-123'
-      server: ${{ secrets.SERVER }}
+      project: 'MyProject'
```

Comment:
we should change the readme to show the api_key and project being used as an ENV instead of a tag

Reply:
done